server/pkg/config/config.go

```diff
@@ -0,0 +1,113 @@
+package config
+
+import (
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+var (
+	Cfg = new(Config)
+)
+
+type (
+	Config struct {
+		Web     Web    `yaml:"web"`
+	}
+
+	Web struct {
+		TaskIdPath  string `yaml:"task_id_path"`
+		UploadDir  string `yaml:"upload_dir"`
+		TasksDir  string `yaml:"tasks_dir"`
+		SqlitedbFilePath string `yaml:"sqlitedb_file_path"`
+		Address string `yaml:"address"`
+		Port int `yaml:"port"`
```

Comment:
Are these all necessary for the user to configure? `TaskIdPath`, `TasksDir` is need to configure?

Reply:
Theses have default values